app/build.gradle

```diff
@@ -66,4 +68,5 @@ dependencies {
     implementation 'com.google.firebase:firebase-analytics-ktx'
     implementation 'com.android.support:multidex:1.0.3'
     implementation 'com.google.android.gms:play-services-ads:19.8.0'
+    implementation 'com.github.luizgrp:SectionedRecyclerViewAdapter:v3.2.0'
```

Comment:
We should be careful about adding third-party packages. As long as it's maintained, we should be ok. I think it is also possible to do this with nested recycler views.